tools/data-handler/src/export-site.ts

```diff
@@ -84,6 +84,7 @@ export class ExportSite extends Export {
     }
     // Use spawnsync to npx execute the program "antora"
     try {
+      // Allthough we use pnpm, only npm is available at runtime
```

Comment:
nom -> npmx ?

Reply:
@samilempinen I assumed that you've packaged it so that it will be installed using `npm install cyberismo -g` from npm or in some similar.

Reply:
Clarified comment